pic32/cores/pic32/WInterrupts.c

```diff
@@ -263,12 +265,22 @@ void __attribute__((nomips16,at_vector(_EXTERNAL_0_VECTOR),interrupt(IPL4SRS)))
 void __attribute__((interrupt(),nomips16)) ExtInt0Handler(void)
 #endif
 {
-
+	VEC_ENT_CODE(EXT_INT0,INT0IF,PIN_INT0);
```

Comment:
Is this the right order for things here? This would give an extra execution of the code after the interrupt pin has returned to high - is that really wanted?

    HIGH - idle
    HIGH - idle
    LOW - trigger (IFS changed to 1)
    LOW - retrigger (IFS remains at 1)
    LOW - retrigger (IFS remains at 1)
    HIGH - retrigger (IFS cleared to 1) <- ISR called even though interrupt pin is now HIGH
    HIGH - idle

I think the desired operation should only call the ISR while the input is low - which means wrapping the call to the user's ISR function in a check of the IO pin as well as the decision to set/clear the IFS flag.